### PR TITLE
release: v0.11.0

### PR DESCRIPTION
**Pull request by jma**
Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

## Why are you opening this PR?

- release 0.11.0

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: RELEASE-NOTES.rst [hunk 1]
```diff
@@ -18,6 +18,176 @@
...
+-  Limits length of document title to 150 characters in detailed view.
+   Adds a *Show more*/*Show less* link if the title is truncated.
+-  Sorts pickup locations alphabetically when placing a request for an
+   item.
+
+Professional interface
+~~~~~~~~~~~~~~~~~~~~~~
+
+-  Adds inventory list functionnality and view based on ``item``
+   resources. The can acces them using the ``Admin`` menu. This
```
Comment (benerken) [inline RELEASE-NOTES.rst:45]: Line 45: please change 
The can acces them using the ``Admin`` menu.
by
The librarian can acces them using the ``Admin`` menu.

Comment (pronguen) [inline RELEASE-NOTES.rst:45]:
```suggestion
   resources. The librarian can access them using the ``Admin`` menu. This
```

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -18,6 +18,176 @@
...
+-  Adds missing translations of item notes types.
+-  Limits length of document title to 150 characters in detailed view.
+   Adds a *Show more*/*Show less* link if the title is truncated.
+-  Sorts pickup locations alphabetically when placing a request for an
+   item.
+
+Professional interface
+~~~~~~~~~~~~~~~~~~~~~~
+
+-  Adds inventory list functionnality and view based on ``item``
```
Comment (pronguen) [inline RELEASE-NOTES.rst:44]:
```suggestion
-  Adds inventory list functionality and view based on ``item``
```

File: RELEASE-NOTES.rst [hunk 3]
```diff
@@ -18,6 +18,176 @@
...
+   resources. The can acces them using the ``Admin`` menu. This
+   functionnality allows the librarian to display a list of items,
```
Comment (pronguen) [inline RELEASE-NOTES.rst:46]:
```suggestion
   functionality allows the librarian to display a list of items,
```

File: RELEASE-NOTES.rst [hunk 4]
```diff
@@ -18,6 +18,176 @@
...
+-  Splits document ``JSONSchema`` into smaller files, to improve
+   readability. JSON references are resolved on the fly.
+
+API
+---
+
+-  Refactors the permission processe using a permission factory and
+   classes for specific resources instead of all resources.
+
+   -  Refactors permission factories for resources: organisation,
```
Comment (pronguen) [inline RELEASE-NOTES.rst:87]: There is no resource called "acquisition"

File: RELEASE-NOTES.rst [hunk 5]
```diff
@@ -18,6 +18,176 @@
...
+
+-  Adds an ``id`` on all menus in order to simplify and improve Cypress
+   tests.
+
+   -  Replaces ``getId()`` by ``idAttribute`` pipe from Angular.
+   -  Doesn’t hide the Debug toolbar in Cypress tests as ``FLASK_DEBUG``
+      should be set to ``False`` when launching the server.
+   -  Creates new ``setLanguageToEnglish`` Cypress command to set
+      language to English.
+   -  Deletes all ``cy.visit()`` method and use menus to navigate in the
```
Comment (pronguen) [inline RELEASE-NOTES.rst:114]:
```suggestion
   -  Deletes all ``cy.visit()`` methods and use menus to navigate in the
```

File: RELEASE-NOTES.rst [hunk 6]
```diff
@@ -18,6 +18,176 @@
...
+      results relevance.
+
+-  Adds a ``second_call_number`` field to ``item`` resource.
+-  Splits document ``JSONSchema`` into smaller files, to improve
+   readability. JSON references are resolved on the fly.
+
+API
+---
+
+-  Refactors the permission processe using a permission factory and
```
Comment (pronguen) [inline RELEASE-NOTES.rst:84]:
```suggestion
-  Refactors the permission processes using a permission factory and
```

File: RELEASE-NOTES.rst [hunk 7]
```diff
@@ -18,6 +18,176 @@
...
+   and adds a triple slash directive on 2 JS files (from Cypress).
+
+Fixed issues
+------------
+
+-  `#880`_: Reduce size of title in document detailed view
+-  `#882`_: Translations of actions realised in circulation UI
+-  `#883`_: Improvement needed on the request information when doing a
+   checkin
+-  `#886`_: Clear the patron info on top of checkin form when quiting it
```
Comment (pronguen) [inline RELEASE-NOTES.rst:161]:
```suggestion
-  `#886`_: Clear the patron info on top of checkin form when quitting it
```

**Pull request merged**